src/Bridge/Symfony/Form/Type/EnumType.php

```diff
@@ -14,6 +14,14 @@
  */
 final class EnumType extends AbstractType
 {
+    /** @var bool */
+    private $classPrefixed;
+
+    public function __construct($classPrefixed = null)
+    {
+        $this->classPrefixed = !is_null($classPrefixed) ? $classPrefixed : false;
```

Comment:
This should probably be cast to bool. Or I should migrate this to php 7 ^^

Reply:
What do you think about `is_null($classPrefixed) ? false : $classPrefixed;`? One less operator, but does it still feel as clear?

Reply:
Ok then stick with it, it's good like that.

Reply:
Why not simply passing a `bool` to the constructor?

Reply:
master does not support php 7 (yet)

Reply:
I talking about `$classPrefixed = false` instead of `null`.

> master does not support php 7 (yet)

Then, you can use this: https://packagist.org/packages/webmozart/assert

Reply:
Oh right, makes sense!